src/MetaTags/Image.php

```diff
@@ -65,4 +65,22 @@ private function get_data( $id ): array {
 			'alt'    => get_post_meta( $id, '_wp_attachment_image_alt', true ) ?: get_the_title( $id ),
 		] : [];
 	}
+
+	private function get_from_post_meta( $meta_value ) {
+		if ( ! filter_var( $meta_value, FILTER_VALIDATE_URL ) ) {
+			$meta_value = Helper::render( $meta_value );
+		}
+		return $this->get_data_from_url( $meta_value );
+	}
+
+	private function get_from_settings(): array {
+		$option = get_option( 'slim_seo', [] );
+		$post_type_object = get_queried_object();
```

Comment:
Chỗ này dùng cho singular, nên tên biến ko phải là `$post_type_object` nhé, mà là `$post`. Và ko dùng `get_queried_object()` nhé, dùng `$this->get_queried_object()` vì queried object có thể bị thay đổi. Xem `Context` để biết nhé.